dubbo-rpc/dubbo-rpc-triple/src/main/java/org/apache/dubbo/rpc/protocol/tri/rest/mapping/DefaultRequestMappingRegistry.java

```diff
@@ -190,6 +215,9 @@ public HandlerMeta lookup(HttpRequest request) {
 
         Candidate winner = candidates.get(0);
         RequestMapping mapping = winner.mapping;
+
+        processCors(mapping, request, response);
```

Comment:
Implementing an extension for org.apache.dubbo.rpc.protocol.tri.rest.filter.RestHeaderFilterAdapter and move logic into it would likely be more appropriate(I didn't think about it before).